portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeSiteNav.vm

```diff
@@ -169,6 +171,18 @@
 
             document.querySelector('.Mrphs-portalWrapper').focus();
         }, 50);
+
+        const siteDescPopoverList = Array.prototype.slice.call(document.querySelectorAll('[data-bs-toggle="popover"]'));
+        siteDescPopoverList.map((el) => {
```

Comment:
map isn't the right thing to use here, imo. Map returns a new array with the mapped elements in it, which you don't need here. I'd just use forEach.